docs/github-release-notes.md

```diff
@@ -0,0 +1,17 @@
+# Overview
+These standards are in place to make release notes standard across the programme so that changes can be easily identified including new features and breaking changes.
+
+# Github Release Note Standards
+## Title
+Version
+## Description 
+  ### Major
+  list of bullet major changes
+
+  ### Minor
+  list of bullet minor changes
+
+  ### Patch
+  list of bullet patch changes
```

Comment:
I think it might be helpful here to spell out what criteria a major, minor or patch change have, possibly with some examples. Linking to the semver definition may be useful as well.

Reply:
I have added some detail to the overview section

Reply:
We already have a page in the development guide that explains semver.  I'm not keen on duplicating that content here too.

Instead it might be be better to reference that we're using semantic versioning and reference the existing page in the development guide.  https://github.com/DEFRA/ffc-development-guide/blob/master/docs/version-control.md